README.md

```diff
@@ -517,6 +517,13 @@ __::isCollection([1, 2, 3]);
 // >> true
 ```
 
+##### [__::isCollection](src/__/objects/isEqual.php)
+Performs a deep comparison between two values to determine if they are equivalent.
+```php
+__::isEqual([1, 1.0]);
```

Comment:
I think you might be missing one argument from this example, there's only one array